ui/src/album/AlbumActions.js

```diff
@@ -28,6 +29,7 @@ const AlbumActions = ({
   const dispatch = useDispatch()
   const translate = useTranslate()
   const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'))
+  const isSmall = useMediaQuery((theme) => theme.breakpoints.up('sm'))
```

Comment:
This would be "Not Small", right? Small would be `down('sm')`. This causes confusion when reading things like
`{isSmall && <ToggleFieldsMenu resource="albumSong" />}` when it really means "Show the menu if it is NOT small".